kafka/consumer/group.py

```diff
@@ -552,11 +552,8 @@ def committed(self, partition):
                 committed = None
         return committed
 
-    def topics(self):
-        """Get all topics the user is authorized to view.
-
-        Returns:
-            set: topics
+    def _fetch_all_topic_metadata(self):
```

Comment:
Is the Java client structured this way with this private method?

It's a nice clean way to do it this way, but just curious how they handle it...

Reply:
Yep! I think that's what I based it on https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java#L1892